cpu/sam0_common/periph/spi.c

```diff
@@ -142,10 +147,6 @@ int spi_acquire(spi_t bus, spi_cs_t cs, spi_mode_t mode, spi_clk_t clk)
 
 void spi_release(spi_t bus)
 {
-    /* disable device and put it back to sleep */
-    dev(bus)->CTRLA.reg &= ~(SERCOM_SPI_CTRLA_ENABLE);
-    while (dev(bus)->SYNCBUSY.reg & SERCOM_SPI_SYNCBUSY_ENABLE) {}
-    poweroff(bus);
```

Comment:
Furthermore if you put the first two lines back in (clearing ENABLE) but not the poweroff(bus), the power consumption goes high. So I don't think it is a poweroff() happening too soon after disable.